sample-bundle-scr/src/main/resources/OSGI-INF/serviceComponents.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is manually generated as the [maven SCR plugin](https://mvnrepository.com/artifact/org.apache.felix/maven-scr-plugin/1.26.4), to autogenerate it reading the project annotations, is not compatible with classes built with Java15.